policies/bundles/DocumentCSAFFilenameValid/metric.rego

```diff
@@ -0,0 +1,58 @@
+package clouditor.metrics.document_csaf_filename_valid
+
+import rego.v1
+import input as document
+
+default applicable := false
+
+default compliant := false
+
+applicable if {
+	# check resource type
+    resourceTypeValid
+}
+
+# Check if the resource type contains "ServiceMetadataDocument" or "SecurityAdvisoryDocument"
+resourceTypeValid if {
+	"ServiceMetadataDocument" in document.type
+}
+
+resourceTypeValid if {
+	"SecurityAdvisoryDocument" in document.type
+}
+
+compliant if {
+	path = document.path
+	is_string(path)
+
+	# Check if "/document/tracking/id" is lower case
```

Comment:
That is not the path. We have to check the Id, which should be the document identifier.